content/quickstart/dotnet/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "C#"
+date: 2018-07-16T14:29:15-07:00
```

Comment:
Please update this date or were you purposefully using this date to support listing C# after C++?
If so please instead use the `weight` attribute but really don't worry about that, Erlang is listed as the top so please use the date `date: 2019-01-07T00:01:21-07:00`

Reply:
What date should I put here? Currently I put a random date.

Reply:
Please use `date: 2019-01-07T00:01:21-07:00` and I don't know if you saw this but I had commented with it before :)
<img width="662" alt="screen shot 2019-01-08 at 6 43 43 am" src="https://user-images.githubusercontent.com/4898263/50808692-ce1e2f80-1310-11e9-91a2-8ca5c224ba7a.png">

Reply:
Thanks. I applied your date. Just curious - is there any rule on how the date should be set here in general?

Reply:
Just the current date for which you made a change.